src/simple_treeview.rs

```diff
@@ -0,0 +1,72 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a TreeView with either a ListStore or TreeStore.
+
+extern crate glib;
+extern crate gtk;
+
+use gtk::prelude::*;
+use gtk::{
+    CellRendererText, TreeStore, TreeView, TreeViewColumn,
+    Window, WindowPosition, WindowType
+};
+
+fn append_column(tree: &TreeView, id: i32) {
+    let column = TreeViewColumn::new();
+    let cell = CellRendererText::new();
+
+    column.pack_start(&cell, true);
+    column.add_attribute(&cell, "text", id);
+    tree.append_column(&column);
+}
+
+fn main() {
+    if gtk::init().is_err() {
+        println!("Failed to initialize GTK.");
+        return;
+    }
+
+    let window = Window::new(WindowType::Toplevel);
+
+    window.set_title("Simple TreeView example");
+    window.set_position(WindowPosition::Center);
+
+    window.connect_delete_event(|_, _| {
+        gtk::main_quit();
+        Inhibit(false)
+    });
+
+    // we create a view
+    let tree = TreeView::new();
+    // we create a model with two rows
+    let store = TreeStore::new(&[String::static_type(), i32::static_type()]);
+
+    // when a line is selected, this signal is called
+    tree.connect_cursor_changed(move |tree_view| {
+        let selection = tree_view.get_selection();
+        if let Some((model, iter)) = selection.get_selected() {
+            println!("Hello '{}' from line {}",
+                     model.get_value(&iter, 0).get::<String>().unwrap(),
+                     model.get_value(&iter, 1).get::<i32>().unwrap() + 1);
+        }
```

Comment:
I wanted to link from the tutorial to our examples. But that's well saw, I'll add comments!